test/bootstrap.php

```diff
@@ -2,11 +2,13 @@
 
 declare(strict_types=1);
 
-require dirname(dirname(dirname(__DIR__))) . '/bootstrap.php';
+//require dirname(__DIR__, 3). '/bootstrap.php';
```

Comment:
Cool to see the static analysis actually work now on github!

1. Maybe try to configure phpstan at a level that satisfies most important things we want enforced in our project, but low enough to not nag about every nit it finds?
2. And maybe if time allows, address some more of the findings PHPStan now comes up with. A green build would be awesome on `main`.